backend/src/main/java/com/cruru/member/service/MemberService.java

```diff
@@ -23,7 +29,17 @@ public Member create(MemberCreateRequest request) {
             throw new MemberEmailDuplicatedException();
         }
 
-        return memberRepository.save(new Member(request.email(), request.password(), request.phone()));
+        String encodedPassword = generateEncodedPassword(request);
+        Member newMember = new Member(request.email(), encodedPassword, request.phone());
+        return memberRepository.save(newMember);
+    }
+
+    private String generateEncodedPassword(MemberCreateRequest request) {
+        String rawPassword = request.password();
+        if (rawPassword.length() < PASSWORD_MIN_LENGTH || rawPassword.length() > PASSWORD_MAX_LENGTH) {
+            throw new MemberPasswordLengthException(PASSWORD_MIN_LENGTH, rawPassword.length());
```

Comment:
```suggestion
            throw new MemberPasswordLengthException(PASSWORD_MAX_LENGTH, rawPassword.length());
```
인자 명은 max인데 min이 입력되고 있는 것 같네요.